packages/cli/src/commonlib/log.ts

```diff
@@ -46,50 +46,46 @@ export class CLILogProvider implements LogProvider {
     return path.join(os.tmpdir(), `.${ConfigFolderName}`, "cli-log", this.logFileName);
   }
 
-  trace(message: string): Promise<boolean> {
-    return this.log(LogLevel.Trace, message);
+  verbose(message: string): void {
```

Comment:
Is there more detailed reason on changing `Promise` to `void`?
IMO, `Promise` is more extensible than `void`, and changing `Promise` to `void` may have potential performance downgrade (e.g., async file operation to sync one)